hack/gen-client-python/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $(dirname "$0")/../../hack/common.sh
```

Comment:
We're making (reasonable) assumptions about the directory structure layout here anyways. All files fall under the `hack` directory so I think
`source $(dirname "$0")/../common.sh`
is the same thing, right?

Same observation for `generate.sh` of course.